src/dispatch/Dispatch.ts

```diff
@@ -224,6 +224,15 @@ export class Dispatch {
     }
 
     private createRequest(): PutRumEventsRequest {
+        if (this.config.alias) {
```

Comment:
Issue(small): we need to be mindful about the number of lines we add to this package. Pretty sure we can just go ahead and add `Alias` like this. If it's undefined, then it won't be included. 

```suggestion
 private createRequest(): PutRumEventsRequest {
            return {
                BatchId: v4(),
                AppMonitorDetails: this.eventCache.getAppMonitorDetails(),
                UserDetails: this.eventCache.getUserDetails(),
                RumEvents: this.eventCache.getEventBatch(),
                Alias: this.config.alias
            };
        }
```

Reply:
good callout. updated